pipelines/agg_matched.yaml

```diff
@@ -6,20 +6,20 @@ tasks:
       connections.package: validate_drp
       connections.metric: PA1
       connections.agg_name: Mean
-  agg_mean_AM1:
-    class: metric_pipeline_tasks.MatchedCatalogsAggregationTask
-    config:
-      # The special agg_name summary defaults to median.
-      # It can be changed via the config.
-      # This is an example of how to change it.
-      # It must be a valid atribute of the numpy package.
-      connections.package: info
-      connections.metric: AM1
-      connections.agg_name: summary
-      python: |
-        config.agg.summary = 'mean'
+#  agg_mean_AM1t:
```

Comment:
I commented this section out because it was getting a dimensions mismatch when I was testing. I can uncomment this part before merging.

Reply:
Please do.